Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN pip install -r requirements.txt
 
 ADD . /ingest-graph-validator/
 RUN pip install -e .
+RUN pip install -r requirements.txt
```

Comment:
Shouldn't it still be installed with setuptools?

Reply:
You are totally right. I changed `setup.py` and moved every dependency setup there. I tested it and it worked for me.